Makefile

```
@@ -0,0 +1,81 @@
+	@echo "  test                    - Run all tests"
+	@echo "  test-verbose            - Run all tests with verbose output"
+	@echo "  test-coverage           - Run all tests with coverage report"
+	@echo "  test-coverage-html      - Run all tests and generate HTML coverage report"
+	@echo "  test-clean              - Clean test cache and run tests"
+	@echo "  install-pre-push-hook   - Install the pre-push git hook"
+	@echo "  uninstall-pre-push-hook - Uninstall the pre-push git hook"
+
+lint:
+	golangci-lint run ./...
+
+test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)